rebar.config

```diff
@@ -10,7 +10,9 @@
     %% Redis Cluster
     {eredis_cluster, {git, "https://github.com/adrienmo/eredis_cluster", {tag, "0.5.12"}}},
     %% Riak KV
-    {riak_client, "~>2.5"}
+    {riak_client, "~>2.5"},
+    %% AQL client
+    {aqlc, {git, "https://github.com/mrshankly/aqlc.git", {branch, "master"}}}
```

Comment:
Are you planning on tagging a particular version of the AQL Client or publishing it on hex.pm?  I would rather reference a stable release than the master branch.

Periodically I try to update dependencies to their latest version, and typically I try out different patch and minor versions. If for some reason you wish to add breaking changes, this will break FMKe's version of aqlc down the line. Another thing this helps is to lock down the versions of the dependencies of `aqlc`.

If you need some help for this I'd be happy to walk you through the release process 😄

Reply:
I've published it on hex.pm, I've updated the `rebar.config` file to use that package.